docs/lab/README.md

```diff
@@ -0,0 +1,93 @@
+# Kernel Module Management Lab 
+
+Kernel Module Management a.k.a KMM manages out of tree kernel modules in Kubernetes.
+
+By installing the controller in a Kubernetes cluster we will get a new Namespace where a Deployment will run and create a ReplicaSet.
+
+Once the ReplicaSet is running we can create and use a new Module object which will manage the build and/or run of a kernel module.
+
+In this document we will find examples and YAML files including in-cluster module building, device plugins and use of module loaders with [KMM](https://github.com/kubernetes-sigs/kernel-module-management) for different Kubernetes environments : [EKS](aws), [GKE](gce), [Azure](azure), [Ubuntu bare metal](ubuntu). 
+
+## Installation
+
+### Command line:
+
+``kubectl apply -k https://github.com/kubernetes-sigs/kernel-module-management/config/default``
```

Comment:
I would link the [getting-started](https://github.com/ybettan/kernel-module-management#getting-started) doc instead so we have a single place to update if we have deployment changes.